03-viz.Rmd

```diff
@@ -1,40 +1,528 @@
+print(morley_hist)
+```
+
+Wow, impressive! These measurements of the speed of light from 1879 had errors around *0.05%* of the true speed. This shows
+you that even though experiments 2 and 5 were perhaps the most accurate, all of the experiments did quite an 
+admirable job given the technology available at the time period.
+
+
+## Explaining the visualization
+#### *Tell a story* {-#my-section}
+
+Typically, your visualization will not be shown completely on its own, but rather it will be part of a larger presentation.
+Further, visualizations can provide supporting information for any part of a presentation, from opening to conclusion.
+For example, you could use an exploratory visualization in the opening of the presentation to motivate your choice
+of a more detailed data analysis / model, a visualization of the results of your analysis to show what your analysis has uncovered,
+or even one at the end of a presentation to help suggest directions for future work. 
+
+Regardless of where it appears, a good way to discuss your visualization is as a story:
+
+1) Establish the setting and scope, and motivate why you did what you did. 
+2) Pose the question that your visualization answers. Justify why the question is important to answer.
+3) Answer the question using your visualization. Make sure you describe *all* aspects of the visualization (including describing the axes). But you 
+   can emphasize different aspects based on what is important to answering your question:
+    - **trends (lines):** Does a line describe the trend well? If so, the trend is *linear*, and if not, the trend is *nonlinear*. Is the trend increasing, decreasing, or neither?
+                        Is there a periodic oscillation (wiggle) in the trend? Is the trend noisy (does the line "jump around" a lot) or smooth?
+    - **distributions (scatters, histograms):** How spread out is the distribution? Where is it centered, roughly? Are there any obvious "clusters" or "subgroups", which would be visible as multiple bumps in the histogram?  
+    - **distributions of two variables (scatters):** is there a clear / strong relationship between the variables (points fall in a distinct pattern), a weak one (points fall in a pattern but there is some noise), or no discernible
+      relationship (the data are too noisy to make any conclusion)?
+    - **amounts (bars):** How large are the bars relative to one another? Are there patterns in different groups of bars? 
+4) Summarize your findings, and use them to motivate whatever you will discuss next.
+
+Below are two examples of how might one take these four steps in describing the example visualizations that appeared earlier in this chapter.
+Each of the steps is denoted by its numeral in parentheses, e.g. (3).
+
+**Mauna Loa Atmospheric CO2 Measurements:** (1) Many current forms of energy generation and conversion---from automotive engines to natural gas power plants---rely on burning
```

Comment:
You need to re-plot the plot here (easy, given that you made it an object, and here I wouldn't say that you need to show the code to call that object)